docs/platform/longview/longview-app-for-apache/index.md

```diff
@@ -71,46 +101,50 @@ To enable the Apache Longview app manually, follow these steps on your Linode vi
 
 {{< /file >}}
 
-3.  Edit `/etc/linode/longview.d/Apache.conf` to look like the following:
+1.  Edit `/etc/linode/longview.d/Apache.conf` to look like the following:
 
     {{< file "/etc/linode/longview.d/Apache.conf" apache >}}
 location http://127.0.0.1/server-status?auto
 
 {{< /file >}}
 
-4.  Restart Apache:
+1.  Restart Apache:
 
-    Debian and Ubuntu:
+    >**Debian and Ubuntu**:
+    >
+    >     sudo systemctl restart apache2
 
-        service apache2 restart
+    >**Other Distributions**:
+    >
+    >     sudo apachectl restart
 
-    CentOS and Fedora:
+1.  Restart Longview:
 
-        apachectl restart
+    > **CentOS, Debian, and Ubuntu**
+    >
+    >     sudo systemctl restart longview
 
-5.  Restart Longview:
+    >  **Other Distributions**
+    >
+    >     sudo service longview restart
 
-        service longview restart
-
-6.  Refresh the Longview Apache tab in the Linode Manager.
+1.  Refresh the Longview in your Linode Cloud Manager.
 
 You should now be able to see Longview data for Apache. If that's not the case, proceed to the [Troubleshooting](#troubleshooting) section at the end of this article.
 
 ## Viewing Statistics
 
 To see the output for the Longview Apache App:
 
-1.  Log in to the [Linode Manager](https://manager.linode.com/).
-2.  Select the **Longview** tab.
+1.  Log in to the [Linode Cloud Manager](https://cloud.linode.com/).
+2.  Select the **Longview** sidebar.
```

Comment:
Select the **Longview** link in the sidebar.

Reply:
fixed